src/instructlab/sdg/__init__.py

```diff
@@ -1,3 +1,13 @@
 # SPDX-License-Identifier: Apache-2.0
+
+# NOTE: This package imports Torch and other heavy packages.
+__all__ = (
+    "GenerateException",
+    "SDG",
+    "generate_data",
```

Comment:
You can't create an SDG without a Pipeline, and you can't create a Pipeline without a Flow, so I'd add

```
pipeline.Pipeline
default_flows.Flow
default_flows.SimpleKnowledgeFlow 
default_flows.SimpleFreeformSkillFlow
default_flows.SimpleGroundedSkillFlow
default_flows.MMLUBenchFlow
default_flows.SynthKnowledgeFlow
default_flows.SynthSkillsFlow
default_flows.SynthGroundedSkillsFlow
```

We're expecting people to develop custom flows too, which will mean exposing Blocks too, but that can wait